### PR TITLE
Reverts all the emergency meeting PRs.

### PR DESCRIPTION
**Pull request by KathyRyals**
## About The Pull Request

Reverts al three PRs related to the emergency meeting.

## Why It's Good For The Game

This feature is not fun and only adds grief potential to the game.

## Changelog
:cl:
del: Removes the emergency meeting. We sussed out all the imposters.
/:cl:

### CODE REVIEW

**Comment by actioninja** [comment]
It's a fucking april fool's feature lmao

**Comment by InsaneRed** [comment]
its not funny after you get emergency meeting called and thne bombed the 5th time

**Comment by InsaneRed** [comment]
amazing

**Comment by Pinkufeck** [comment]
Still fucking ruins the game for 5 days

**Comment by Qustinnus** [comment]
owned lol

**Comment by actioninja** [comment]
Oh no I got grief in spacemans for 5 whole days 😢

**Comment by Timberpoes** [comment]
I approve of this PR staying open. Code review pending. This PR has been a bane to the admin team and a bane to players. It's funny exactly once, where it is massively funny.

Every time they experience it after that, many of the players we want to keep get closer and closer to wanting to quit the game.

The grief this produces across the servers is asinine. It is frustrating. There only winning move is not to play. It is the antithesis to good natured April Fool's shenanigans.

**Comment by FalkKeegan** [comment]
> 
> 
> I approve of this PR staying open. Code review pending. This PR has been a bane to the admin team and a bane to players. It's funny exactly once, where it is massively funny.
> 
> Every time they experience it after that, many of the players we want to keep get closer and closer to wanting to quit the game.
> 
> The grief this produces across the servers is asinine. It is frustrating. There only winning move is not to play. It is the antithesis to good natured April Fool's shenanigans.

Perfectly describes everything april's fools thrown at sybil.

**Comment by Qustinnus** [comment]
sounds like an IC issue, timber.

File: code/controllers/subsystem/communications.dm [hunk 1]
```diff
@@ -1,67 +1,33 @@
...
 	if(is_silicon)
 		minor_announce(html_decode(input),"[user.name] Announces:")
-		COOLDOWN_START(src, silicon_message_cooldown, COMMUNICATION_COOLDOWN_AI)
+		silicon_message_cooldown = world.time + COMMUNICATION_COOLDOWN_AI
 	else
 		priority_announce(html_decode(user.treat_message(input)), null, 'sound/misc/announce.ogg', "Captain", has_important_message = TRUE)
-		COOLDOWN_START(src, nonsilicon_message_cooldown, COMMUNICATION_COOLDOWN)
+		nonsilicon_message_cooldown = world.time + COMMUNICATION_COOLDOWN
 	user.log_talk(input, LOG_SAY, tag="priority announcement")
 	message_admins("[ADMIN_LOOKUPFLW(user)] has made a priority announcement.")
```
Comment (Fikou) [inline code/controllers/subsystem/communications.dm:29]: other than the emergency meeting stuff these were all code improvements, they shouldnt be reverted

**Review by Fikou — CHANGES_REQUESTED** [review]

**Comment by zxaber** [comment]
The emergency meeting was a funny April Fools event but not one that needs a permanent home. Amogus memes *may* be topical this time next year, but I doubt it goes much further.

If we're doing funny dumb reference PRs for April Fools (and we might as well), we should leave the space open for relevant jokes of the prior year and not ancient jokes about a then-forgotten game.

File: code/controllers/subsystem/communications.dm (hunk 1 above)
Comment (LemonInTheDark) [reply code/controllers/subsystem/communications.dm:29]: stan fikou

**Comment by RaveRadbury** [comment]
> If we're doing funny dumb reference PRs for April Fools (and we might as well), we should leave the space open for relevant jokes of the prior year and not ancient jokes about a then-forgotten game.

In a world where we still have rickrolls you're really saying that a global phenomenon that just happens to be incredibly similar to the game that we all play and work on is going to be _forgotten_ in a year?

**Comment by MrDoomBringer** [comment]
bruh just destroy the comm console lmao

**Comment by maxymax13** [comment]
:trollface:

**Comment by optimumtact** [comment]
good to see the admins realising what it's like when they run events

**Comment by Nebulacrity** [comment]
Based.

**Comment by Mothblocks** [comment]
April fools! 🤡

**Comment by optimumtact** [comment]
> 
> 
> > I approve of this PR staying open. Code review pending. This PR has been a bane to the admin team and a bane to players. It's funny exactly once, where it is massively funny.
> > Every time they experience it after that, many of the players we want to keep get closer and closer to wanting to quit the game.
> > The grief this produces across the servers is asinine. It is frustrating. There only winning move is not to play. It is the antithesis to good natured April Fool's shenanigans.
> 
> Perfectly describes everything april's fools thrown at sybil.

April fools is a stupid thing and I wish we would not do it at all

**Pull request closed without merging**